hls4ml/converters/pytorch_to_hls.py

```diff
@@ -159,6 +160,23 @@ def parse_pytorch_model(config, verbose=True):
 
     n_inputs = 0
 
+    # check for constant nodes
+    merge_layers = ['add', 'mul', 'sub', 'fmin', 'fmax']
+    i = 0  # count number of consts and use it in the name
+    for node in traced_model.graph.nodes:
+        if node.name.split("_")[0] in merge_layers:
```

Comment:
I think this is one of those " instead of ' cases.

Reply:
Right, fixed.